popup/popup.js

```diff
@@ -18,10 +18,12 @@
             liveServerUrl: liveServerAddress.value || ''
         }
 
-        chrome.runtime.sendMessage({
-            req: 'set-live-server-config',
-            data: formData
-        });
+        if (chrome && chrome.runtime) {
```

Comment:
same. is this required? I mean the null check?

Reply:
`chome.runtime` was also showing `undefined`. I thought it's better to have a null check there too.